doc/persistence.rst

```diff
@@ -79,6 +92,16 @@ are 'gzip', 'bz2', 'lzma' and 'xz':
 
     Lzma and Xz compression methods are only available for python versions >= 3.3.
 
+Compressor files provided by the python standard library can also be used to
+compress pickle in a context manager, e.g `gzip.GzipFile`, `bz2.BZ2File`,
```

Comment:
Remove "in a context manager".

Also singe back-tick does italic in rst. You have to use double back-tick to get fixed width font, if that is what you were after.